modules/ossm-config-control-plane-infrastructure-node.adoc

```diff
@@ -0,0 +1,97 @@
+
+.Procedure
+
+. Edit the `ServiceMeshControlPlane` resource to present the YAML file in the editor. In this example, `<name>` represents the name of the `ServiceMeshControlPlane` resource.
++
+[source,terminal]
+----
+$ oc -n istio-system edit smcp <name>
+----
+
+. To run the Istiod component on infrastructure nodes add the `nodeSelector` and `tolerations` to `spec.runtime.components.pilot.pod` in the `ServiceMeshControlPlane` resource.
++
+[source,yaml]
+----
+spec:
+  runtime:
+    components:
+      pilot:
+        pod:
+          nodeSelector: <1>
+            node-role.kubernetes.io/infra: ""
+          tolerations: <2>
+          - effect: NoSchedule
+            key: node-role.kubernetes.io/infra
+            value: reserved
+          - effect: NoExecute
+            key: node-role.kubernetes.io/infra
+            value: reserved
+----
+<1> Ensures that the Istiod pod is only scheduled on an infrastructure node.
+<2> Ensures that the pod is accepted by the infrastructure node.
+
+. To run Ingress and Egress Gateways on infrastructure nodes add the `nodeSelector` and `tolerations` to `spec.gateways.ingress.runtime.pod` and `spec.gateways.egress.runtime.pod` in the `ServiceMeshControlPlane` resource.
++
+[source,yaml]
+----
+spec:
+  gateways:
+    ingress:
+      runtime:
+        pod:
+          nodeSelector: <1>
+            node-role.kubernetes.io/infra: ""
+          tolerations: <2>
+          - effect: NoSchedule
+            key: node-role.kubernetes.io/infra
+            value: reserved
+          - effect: NoExecute
+            key: node-role.kubernetes.io/infra
+            value: reserved
+    egress:
+      runtime:
+        pod:
+          nodeSelector: <1>
+            node-role.kubernetes.io/infra: ""
+          tolerations: <2>
+          - effect: NoSchedule
+            key: node-role.kubernetes.io/infra
+            value: reserved
+          - effect: NoExecute
+            key: node-role.kubernetes.io/infra
+            value: reserved
+
+----
+<1> Ensures that the gateway pod is only scheduled on an infrastructure node
+<2> Ensures that the pod is accepted by the infrastructure node.
+
+. Alternatively, to run all of the Service Mesh components deployed by the SMCP (including Istiod, Ingress and Egress Gateways) along with optional elements (such as Prometheus, Grafana, and Distributed Tracing) on infrastructure nodes add `nodeSelector` and `tolerations` to `spec.runtime.defaults.pod` in the `ServiceMeshControlPlane` resource.
++
+[source,yaml]
+----
+spec:
+  runtime:
+    defaults:
+      pod:
+        nodeSelector: <1>
+          node-role.kubernetes.io/infra: ""
+        tolerations: <2>
+        - effect: NoSchedule
+          key: node-role.kubernetes.io/infra
+          value: reserved
+        - effect: NoExecute
+          key: node-role.kubernetes.io/infra
+          value: reserved
+
+----
+<1> Ensures that the gateway pod is only scheduled on an infrastructure node
```

Comment:
Hi, should say here something: 
`Ensures that the smcp pods are only scheduled on an infrastructure node, including optional elements`

Reply:
done